apps/user/templates/user/profile.html

```diff
@@ -1,39 +1,42 @@
 {% extends "base.html" %}
 
 {% load static %}
+{% load i18n %}
 {% load webpack_loader %}
 
 {% block content %}
 
 <div class="container-fluid">
-  <h1 class="title">Welcome, {{ user.first_name }}!</h1>
+  <h1 class="title">{% blocktranslate with firstname=user.first_name %}Welcome, {{ firstname }}!{% endblocktranslate %}
+  </h1>
   <hr>
-  Name: {{user.first_name}} <br>
-  Lastname: {{user.last_name}}<br>
-  Joined on: {{user.date_joined}}<br>
+  {% blocktranslate with firstname=user.first_name %}Name: {{ firstname }}{% endblocktranslate %} <br>
+  {% blocktranslate with lastname=user.last_name %}Lastname: {{ lastname }}{% endblocktranslate %}<br>
+  {% blocktranslate with date=user.date_joined %}Joined on: {{date}}{% endblocktranslate %}<br>
+  <br>
```

Comment:
Could those be just {% translate %} blocks here?

Reply:
> Could those be just {% translate %} blocks here?

Not if there's a placeholder in a string.

Reply:
Switched to {% translate %}  like so:
`{% translate "Name:" %} {{ user.first_name }}`